docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Browser OpenTelemetry
+slug: opentelemetry
+createdAt: 2024-08-23T22:55:19.000Z
+updatedAt: 2024-08-23T22:55:19.000Z
+---
+
+Highlight's JavaScript SDK offers built-in support for collecting OpenTelemetry data from client-side applications, allowing you to seamlessly integrate OpenTelemetry tracing into your web applications.
+
+## Enabling OpenTelemetry with Highlight
+
+To enable OpenTelemetry tracing in your application using Highlight's SDK, set the `enableOtelTracing` configuration option when initializing the SDK:
+
+```ts
+H.init({
+  // ...
+  enableOtelTracing: true
+})
```

Comment:
should we update the sdk docs on this config? or are we keeping that for another pr

Reply:
Yeah, probably a good idea to advertise that at this point!